### DISCUSSION TITLE
ユーザーを管理するAPIについて

### DISCUSSION THREAD

**Discussion by Derorin719** [growilabs/growi#8087] [Help / Questions] [answered]
ユーザーを新規作成したり、ユーザー情報を編集したりするAPIはありますか？

**Comment by yuki-takei** [comment, marked as answer]
API 呼び出しのみで手軽に使えるものは現状の最新版 v6.2.1 では存在しません。

一応、`/_api/v3/users/invite` 呼び出しでユーザーの作成だけはできます。
https://docs.growi.org/redoc.html#operation/inviteUser

ただし、作成したユーザーはすぐ利用できる状態にはなっていません。(ログインが必要)
また、当該 REST endpoint は `/me` で予め作成できる access token には対応していないため、Admin 権限をもつユーザーとしてログイン済みのセッション情報を利用可能な cookie とともに呼び出す必要があります。
